src/Nowruz/modules/feeds/createPostModal/useCreatePostModal.tsx

```diff
@@ -21,12 +21,22 @@ const schema = yup
         value: yup.string(),
       }),
     content: yup.string().required('Content is required'),
-    file: yup.mixed().nullable().required('You should select an Image'),
+    file: yup
+      .mixed()
+      .nullable()
+      .required('You should select an Image')
```

Comment:
What is the point of nullable and required together? Does it mean that just 'undefined' isn't accepted?